app/javascript/packs/src/settings/app.jsx

```diff
@@ -1,22 +1,40 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import { BrowserRouter, Route, Switch, Redirect } from "react-router-dom";
 import { Provider } from "react-redux";
-import { IntlProvider } from "react-intl";
-import { addLocaleData } from "react-intl";
+import { IntlProvider, addLocaleData } from "react-intl";
 import enLocaleData from "react-intl/locale-data/en";
+
 import { flattenMessages } from "../../locales/utils";
 import store from "../../app/store";
 import messages from "../../locales/messages";
 
 import Navigation from "../../shared/navigation";
+import NotFound from "../../shared/404/NotFound";
+import MainNav from "./components/MainNav";
+import SettingsAccount from "./components/account/SettingsAccount";
+import SettingsTeams from "./components/team/SettingsTeams";
 
 addLocaleData([...enLocaleData]);
 const locale = "en-US";
 
 const SettingsPage = () =>
   <div>
     <Navigation page="Settings" />
-    ....
+    <div className="container">
+      <MainNav />
```

Comment:
app.jsx looks much cleaner now that all navigation is in <MainNav /> component :+1: